src/base/utils/foreignapps.cpp

```diff
@@ -186,13 +180,11 @@ namespace
             while (!found && !versions.empty())
             {
                 const QString version = versions.takeLast() + "\\InstallPath";
-                LPWSTR lpSubkey = new WCHAR[version.size() + 1];
-                version.toWCharArray(lpSubkey);
-                lpSubkey[version.size()] = 0;
+                std::wstring lpSubKeyBuffer(version.size() + 5, 0);
+                version.toWCharArray(lpSubKeyBuffer.data());
```

Comment:
FYI, there is `std::wstring QString::toStdWString() const` method.

Reply:
> FYI, there is `std::wstring QString::toStdWString() const` method.

Great